pkg/apis/manager/v1beta1/gen-doc/Dockerfile

```diff
@@ -1,17 +1,15 @@
 FROM pseudomuto/protoc-gen-doc
```

Comment:
I checked to build this image with tags `1.5.1`, the expected error was printed :
![image](https://user-images.githubusercontent.com/37469330/154838233-04df46bc-3527-4c52-be3b-4857a802314e.png)

Also, with tags `1.4.1`, such error was not printed as expected, since it is based on Debian as you mentioned.
![image](https://user-images.githubusercontent.com/37469330/154838299-0bd1b715-173f-446a-b54c-1f3381d7ee17.png)

What do you think of whether is it necessary to specify the exact tag here for possible future update on this image or not?

Refs: https://hub.docker.com/r/pseudomuto/protoc-gen-doc/tags

Reply:
As you say, we have to specify the image tag in images used by end users, but our `protoc-gen-doc` image is only used by developers.

So, I think it might be better not to specify a tag for `pseudomuto/protoc-gen-doc` since specifying the exact tag increases the management cost.

What do you think @kubeflow/wg-automl-leads @anencore94 ?

Reply:
I agree with @tenzen-y :) The potential management const is unnecessary for this docker image. Since we always check the validation with CI process already.

Reply:
@anencore94  Do we have this validation in CI process ?

Reply:
We check from make check(which includes make generate) from go test